project-charter.md

```diff
@@ -20,25 +20,25 @@ The mission of the Project is to:
 
 # **2. Governance Board**
 
-1.  The Governance Board (the “GB”) shall be responsible for overall oversight of the Project and coordination of the efforts of any Technical Committees.
+1.  The Governance Board (the “GB”) shall be responsible for overall oversight of the Project and coordination of the efforts of any Technical Committees and Workgroups.
```

Comment:
Up to now we were using "working group" instead of "workgroup". I'm not sure there is a difference, though. I'm sure you know better, so you decide which one to use ;-)

Reply:
I agree that either word works. I prefer "workgroup" because it is succinct, only one word, and it feels more natural.

Reply:
OK from my side.

Reply:
D&I WG had a discussion about Workgroup vs. Working Group. After looking up the terms, I am still mostly indifferent but I see a trend towards Working Group as an industry standard.

I seem to be the only one who prefers Workgroup and propose to change it to Working Group.

Reply:
Resolving, since we seem to have consensus...